src/nanoemoji/nanoemoji.py

```diff
@@ -270,16 +272,30 @@ def _draw_glyph_extents(ufo: ufoLib2.Font, glyph: Glyph):
 
 
 def _glyf_ufo(ufo, color_glyphs):
```

Comment:
while working on #52 I came across the fact that "glyf" as a `--color_format` and `_glyf_ufo` as function name in here don't make much sense when --output_format is .otf (i.e. CFF-flavored OpenType). The name `glyf` is tied to TrueType. We want to convey the idea that this is _not_ a layered font (like COLR is), and that it is _not_ colored as it only has black & white (or if you like, only black). How about nouns like "flat", or "black"? Or even just "simple" or "normal"..

Reply:
Migrating to a separate issue.